sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -172,10 +173,14 @@ private void performWorkload(BaseSubscriber<Object> documentSubscriber, Operatio
             RequestOptions options = new RequestOptions();
             String partitionKeyValue = docsToRead.get(container.getId()).get(index).getId();
             options.setPartitionKey(new PartitionKey(partitionKeyValue));
-            obs = container.readItem(docsToRead.get(container.getId()).get(index).getId(),
-                new PartitionKey(partitionKeyValue),
-                PojoizedJson.class)
-                .flux();
+
+            Mono sparsitySleepMono = sparsityMono();
```

Comment:
Why only in read operation , don't we need in write and query , as they all part of on CTL workload

Reply:
Yea, make sense Naveen, changed to all operations in CTL worklaod (write,query, read)